### DISCUSSION TITLE
Option for default playback device to be headset

### DISCUSSION THREAD

**Discussion by mikeoh90** [steam3d/MagicPods-Windows#161] [Ideas]
I use my airpods for work for Microsoft Teams and VOIP phone calls and therefore need the headphones to be connected but the 'headset' to be the playback device. When I use MagicPods to connect, it connects great, but as headphones and not headset (so microphone doesnt work). Just wondering if an option to connect as headset by defauly could be an option in the experimental features/general settings please?

**Comment by steam3d** [comment]
Hi. The MagicPods connect both services the headphones and the headset.  Windows will automatically switch headphones to headset when any app tries to use the microphone. Thus, I cannot control this process.  Windows 11 has much better experience with it.

Note. When AirPods uses headset mode the MagicPods immediately disable ear detection and audio endpoint switch options and enable it automatically when AirPods switches to headphones mode.
